src/hotspot/cpu/x86/stubGenerator_x86_64.cpp

```diff
@@ -4009,8 +4009,8 @@ void StubGenerator::generate_initial_stubs() {
   create_control_words();
 
   // Initialize table for unsafe copy memeory check.
-  if (UnsafeCopyMemory::_table == nullptr) {
-    UnsafeCopyMemory::create_table(16);
+  if (UnsafeMemoryAccess::_table == nullptr) {
+    UnsafeMemoryAccess::create_table(26);
```

Comment:
This needs comment

Reply:
I added 10 to the table size because I knew I was going to add 7 places where a mark was required for setMemory.  I left 3 for safety.  The algorithm changed so only 4 are needed.

The algorithm has since changed, so I changed this to:

`UnsafeCopyMemory::create_table(16 + 4); // 16 for copyMemory; 4 for setMemory`

I did a similar change to all other table creation numbers.

Reply:
I took what was there and added 10, since the original algorithm required 7 marks for setMemory. I added 3 for safety. The algorithm changed so only 4 are needed, so I changed this to:

`UnsafeCopyMemory::create_table(16 + 4); // 16 for copyMemory; 4 for setMemory`

Hope this is clearer.